src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationRepositoryTests.cs

```diff
@@ -1028,14 +1007,6 @@ await using (var context = InMemoryApplicationDbContext(contextId))
                 Assert.Single(viewModel.Methodologies);
                 Assert.Equal(methodologyVersion.Id, viewModel.Methodologies[0].Methodology.Id);
                 Assert.Equal(methodologyVersion.AlternativeTitle, viewModel.Methodologies[0].Methodology.Title);
```

Comment:
Going to have some conflicts here with Mark's PR which I've just finished reviewing :)

Reply:
Thanks for the heads up. I've just merged this branch into a temporary branch based on EES-3554 to see all the conflicts. There are in `PublicationServiceTests` and here in `PublicationRepositoryTests` but seem minor to resolve. Happy to pick up them up if EES-3554 is not merged in while @markhiveit is still here.